mlflow/openai/_openai_autolog.py

```diff
@@ -478,3 +487,16 @@ def patched_swarm_run(original, self, *args, **kwargs):
     """
     traced_fn = mlflow.trace(original, span_type=SpanType.AGENT)
     return traced_fn(self, *args, **kwargs)
+
+
+def _generate_model_identity(model_dict) -> int:
```

Comment:
Does `model_dict` only json serializable objects?

Reply:
It could contain non-json serializable objects, I used `default=str` in json.dumps. I don't know how often users pass parameters like `extra_headers`, maybe we could exclude `extra_headers, extra_query, extra_body, timeout` etc. from the dictionary as well 🤔